Finality providers and staking form UI/UX revamp

desamtralized opened the pull request.

desamtralized committed:
feat: new desktop layout for finality providers list

desamtralized committed:
wip: staking dialog

desamtralized committed:
wip: showing staking dialog with new layout and proper data

desamtralized committed:
Refactor staking components and improve form handling

- Update reset prop type from boolean to number
- Implement debounced input validation for StakingTime
- Remove PreviewModal and integrate signing process into main dialog
- Adjust input types and error handling
- Improve form reset and dialog closure behavior

desamtralized committed:
better handling of stake btn while 'loading'

desamtralized committed:
refined staking modal layout

desamtralized committed:
reponsive staking dialog

desamtralized committed:
created general dialog with new layout

desamtralized committed:
general error handling through standard dialog

desamtralized committed:
moving repeated css values to tailwind config

desamtralized committed:
fix: build errors

desamtralized committed:
using default tailwind values in simpledialog

MattNguyen commented on public/icons/wbtc.svg:
oops, probably should replace it with the proper btc svg

https://cryptologos.cc/logos/bitcoin-btc-logo.svg?v=033

desamtralized committed:
Update fonts and refine UI components

- Replace .otf font files with .woff versions
- Add GT America Extended font
- Adjust styling in StakingAmount and Staking components
- Refine UI layout and typography in stake/btc page
- Update coinName to coinSymbol for consistency

desamtralized committed:
fix: style of staking stats cols for mobile

desamtralized committed:
chore: rollback font for now
